ktor-client/ktor-client-features/ktor-client-logging/jvm/src/io/ktor/client/features/logging/LoggerJvm.kt

```diff
@@ -10,3 +10,45 @@ actual val Logger.Companion.DEFAULT: Logger
             delegate.info(message)
         }
     }
+
+
+/***
+ * Workaround Logger to log long messages that would otherwise be truncated in Android.
+ */
+val Logger.Companion.ANDROID: Logger
+    get() = MaxLengthLogger()
+
+
+class MaxLengthLogger(val MAX_INDEX : Int = 4000) : Logger {
```

Comment:
It should be either documented or internal/private instead.

Reply:
I documented it